docs/how-to/h-benchmark.md

```diff
@@ -0,0 +1,107 @@
+# How to benchmark Charmed Apache Kafka
+
+The Apache Kafka benchmark charm uses the OpenMessaging tool to test both producer and consumer performance in the cluster.
```

Comment:
Suggestion: Let's add the link to the benchmark charm somewhere. Otherwise it's rather hard to find, since the charm is not searchable on the charmhub.
```suggestion
[The Apache Kafka benchmark](https://charmhub.io/kafka-benchmark) charm uses the OpenMessaging tool to test both producer and consumer performance in the cluster.
```